submissions/nadsatt/friends-app/modules/img-service.js

```diff
@@ -0,0 +1,49 @@
+export class ImgService {
+    constructor(){
+        this.imgsLoadedCount = 0;
+        this.totalImgs = 400;
+        const femaleNumber = 60;
+        const maleNumber = 40;
+        const usersNumber = 100;
+
+        this.femaleImgPaths = [...Array(femaleNumber).keys()].map(i => `images/female/female${++i}.jpeg`);
+        this.maleImgPaths = [...Array(maleNumber).keys()].map(i => `images/male/male${++i}.jpeg`);
+        this.borderImgPaths = [...Array(usersNumber).keys()].map(() => 'images/user-card-imgs/user-card-border.png');
+        this.backImgPaths = [...Array(usersNumber).keys()].map(() => 'images/user-card-imgs/user-card-back-bg.jpg');
+    }
+
+    loadImgs(){
+        return new Promise((resolve) => {
+            this.userImgs = this.femaleImgPaths.concat(this.maleImgPaths)
```

Comment:
As I told earlier - you relying on order of elements in opaque way. You should avoid such stuff :)